packages/embed/src/factories/create-widget/create-widget.ts

```diff
@@ -64,6 +64,17 @@ export const createWidget = (formId: string, options: WidgetOptions): Widget =>
     )
   }
 
+  if (widgetOptions.autoFocus) {
+    window.addEventListener(
+      'message',
+      getFormReadyHandler(embedId, () => {
+        setTimeout(() => {
```

Comment:
`setTimeout` is necessary because of the loading screen, see https://github.com/Typeform/embed/issues/513#issuecomment-1225593756